benches/README.md

```diff
@@ -0,0 +1,4 @@
+## Benchmarks
+
+Currently, benchmarks following those of [`dashmap`](https://github.com/xacrimon/dashmap/tree/master/benches) and [`hashbrown`](https://github.com/rust-lang/hashbrown/blob/master/benches/bench.rs) are provided. To compare against other hashmap implementations, the benchmarks located in the respective repositories may be executed. Note that `flurry`, like `dashmap`, uses [`criterion`](https://docs.rs/criterion/0.3.1/criterion/) (and [`rayon`](https://docs.rs/rayon/1.3.0/rayon/) for parallel testing), while `hashbrown` uses [`test::bench`](https://doc.rust-lang.org/test/bench/index.html).
```

Comment:
We should probably note here that _our_ port of the `hashbrown` benchmarks still uses `criterion`?

Also, line breaks :p

Reply:
Is this unclear from the last sentence?

Reply:
Ah, interesting. No, I suppose it is clear, I just got confused about what "flurry hashbrown" benchmarks would then look like.